corehq/apps/reports/standard/users/reports.py

```diff
@@ -54,19 +55,21 @@ def get_primary_properties(cls, domain):
         needing to click "See More".
         """
         if domain_has_privilege(domain, privileges.APP_USER_PROFILES):
-            user_data_label = _("Profile or User Data")
+            user_data_label = _("profile or user data")
         else:
-            user_data_label = _("User Data")
+            user_data_label = _("user data")
         return {
-            "username": _("Username"),
-            "email": _("Email"),
-            "domain": _("Project"),
-            "is_active": _("Is Active"),
-            "language": _("Language"),
-            "phone_numbers": _("Phone Numbers"),
-            "location_id": _("Primary Location (mobile users only)"),
+            "username": _("username"),
+            "role_id": _("role"),
```

Comment:
Unfortunately this won't work as expected because role isn't saved in "changes".
Good thing is that it should work after https://github.com/dimagi/commcare-hq/pull/30253/files is done.

I believe that is true for "domain", "is_active", "phone_numbers" and "two_factor_auth_disabled_until" as well.

After the linked PR is deployed we can filter on "change_messages" to check for key
key now -> key later
role -> role
domain -> domain
is_active -> status
phone_numbers -> phone_numbers
location_id -> location
two_factor_auth_disabled_until -> two_factor
(@orangejenny apologies for the miss on my review earlier when these were added)

So it's fine to leave this here, so that it gets updated as well but just flagging that it won't work right now.
Should not be a major change, i am happy to address it as a follow up to the linked PR.

Reply:
Thanks Manish, that's good to know. Just to be clear, once this pr is approved should I wait to merge until your pr is merged?

Reply:
Yes, fine to merge this.